MOBILE_CLIENT/COMMON/_HIGH/TAPJACKING_VULNERABILITY/recommendation.md

```diff
@@ -0,0 +1,73 @@
+
+To mitigate Tapjacking vulnerabilities in mobile applications, developers should:
+
+ * Thoroughly validate and filter user inputs and application settings to prevent exploitation by malicious overlays.
+ * Pay special attention to sensitive settings vulnerable to manipulation by overlays, such as authentication parameters or access controls.
+ * Implement strict input validation checks on user-modifiable settings to prevent unauthorized changes.
+ * Provide clear user guidance within the application on recognizing and avoiding suspicious overlays.
+ * Regularly update the application to address newly identified vulnerabilities.
+ * Consider implementing overlay detection mechanisms to enhance the application's resilience against potential threats.
+ * Set the filterTouchesWhenObscured on views to true.
+ * Set security policies for touch event using onFilterTouchEventForSecurity.
+ * On Android 6 (API level 23) and higher, users have to explicitly grant the permission for the app to create an overlay window.
+ * On Android 12 (API level 31) and higher, apps can pass true into Window.setHideOverlayWindows().
```

Comment:
These mitigations are in the OS itself and not something the user should do, we should just mention them.